splunk_test.go

```diff
@@ -554,6 +554,127 @@ func TestJsonFormat(t *testing.T) {
 	}
 }
 
+// Verify HEC format
+func TestHECFormat(t *testing.T) {
+	hec := NewHTTPEventCollectorMock(t)
+
+	go hec.Serve()
+
+	info := logger.Info{
+		Config: map[string]string{
+			splunkURLKey:                  hec.URL(),
+			splunkTokenKey:                hec.token,
+			splunkFormatKey:               splunkFormatHEC,
+			splunkGzipCompressionKey:      "true",
+			splunkGzipCompressionLevelKey: "1",
+			tagKey:                        "{{.ImageName}}/{{.Name}}",
+			labelsKey:                     "a",
+			envRegexKey:                   "^foo",
+		},
+		ContainerID:        "containeriid",
```

Comment:
Is this a typo? Should it be containerid?

Reply:
It does look like a typo but it is copied from elsewhere in this file (see lines 348 & 452). It actually doesn't matter if it is a typo in this context as it is just test data that isn't used for anything meaningful